Android/Tapp/app/src/main/java/app/anish/com/tapp/utils/ValueContainer.java

```diff
@@ -0,0 +1,25 @@
+package app.anish.com.tapp.utils;
+
+/**
+ * A simple class to hold values of the Generic Type.
+ * Created by anish_khattar25 on 7/25/17.
+ */
+
+public class ValueContainer<T> {
```

Comment:
I need to remove this. I was using this before with some latch countdown stuff I was trying to do but that didn't work out